box-it-script-week-1/box-it.js

```diff
@@ -0,0 +1,86 @@
+// longestStringInArray helper function to determine the width of the box, based on the longest string in the array of strings passed to the boxIt function
+
+function longestStringInArray(arr) {
+    let maxStringLength = 0 
+    for (let element of arr) {
+        if (element.length > maxStringLength) {
+            maxStringLength = element.length;
+        }
+    }
+    return maxStringLength;
+}
+
+
+
+// functions to be used within the main boxIt function
+
+function drawLine(lineLength) {
+    return '━'.repeat(lineLength);
```

Comment:
For best practice move these hard coded strings into variables and reference the variables where needed.

```javascript
const LINE = '━';
const TOP_LEFT_BORDER = '┏';
const TOP_RIGHT_BORDER = `┓`;
...
function drawTopBorder(topBorderLength) {
  return `${TOP_LEFT_BORDER}${drawLine(topBorderLength)}${TOP_RIGHT_BORDER}\n`;
}
```